README.md

```diff
@@ -17,6 +17,8 @@ Knowledge base for DSSG.
 - [MIT: The Analytics Edge](https://www.edx.org/course/the-analytics-edge)
 - [Andrew Ng: Machine Learning](https://www.coursera.org/learn/machine-learning)
 - [CMU: Machine Learning](http://www.cs.cmu.edu/~tom/10701_sp11/lectures.shtml)
+- [Python Data Science Handbook](https://jakevdp.github.io/PythonDataScienceHandbook/)
+- [R for Data Science](https://r4ds.had.co.nz/)
```

Comment:
Should we allow repeated links in different section?

Reply:
Let's have no repeated links, keep the list clean and simple